Tasks/AzureTestPlanV0/Invokers/goinvoker.ts

```diff
@@ -1,15 +1,28 @@
 import tl = require('azure-pipelines-task-lib/task');
 import utils = require('../utils');
 import constants = require('../constants');
+import { executeGo, executeGotestsum } from '../testexecutor';
 
 export async function executeGoTests(testsToBeExecuted: string[]): Promise<number> {
 
-    //Go execution will be added
-    /*executable = go;
-    args = test, ./...  
-    spawn*/
+    let command = constants.GOTESTSUM_INSTALL;
+    let argument = constants.GOTESTSUM_PACKAGE;
+    const installGotestsum = await executeGo(command, argument);
+    console.log(installGotestsum);
 
-    console.log("Go changes1");  
-    return 1;
+    let i = 0;
+    for (let tests of testsToBeExecuted) {
+        try {
+            const modifiedPath = utils.separatePath(tests);
+            const modifiedTest = utils.separateTestName(tests);
+            const command = constants.GOTESTSUM_JUNITFILE;
+            const argument = `TEST-${i}.xml -- ${modifiedPath} -v -run ^${modifiedTest}$ `;
```

Comment:
Let's keep junit file name similar to what we have kept in python like:

'--junitxml=TEST-python-junit.xml']);

here in Go, it can be TEST-Go1-junit.xml or TEST-Go2-junit.xml